home/migrations/0002_users_email.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2 on 2019-05-27 13:51
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('home', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='users',
+            name='email',
+            field=models.CharField(default=django.utils.timezone.now, max_length=100),
```

Comment:
It seems a bit strange to default the `email` field to the current time?

Reply:
Yes, sorry that needed a default value, it will change!